### PR TITLE
sabakan-state-setter: shutdown retired machines

### PR DESCRIPTION
**Pull request by masa213f**
Add a new feature to `sabakan-state-setter` to auto shutdown retired machines.
When this change is released, the retired machines will be shut down at 1:00 AM (JST) every day.

Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

**Review by ymmt2005 — COMMENTED** [review]
LGTM

File: etc/sabakan-state-setter.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+shutdown-schedule: 00 16 * * *
```
Comment (morimoto-cybozu) [inline etc/sabakan-state-setter.yml:1]: Just a piece of trivia:-)
"00" is not common for crontab.  This type of notation sometimes causes errors when used with non-sophisticated libraries.  Especially "08" and "09" can be wrongly considered as illegal octal numbers.
Go's `strconv.Atoi()` is OK.  `robfig/cron` uses `Atoi()` and it's OK too.

Comment (masa213f) [reply etc/sabakan-state-setter.yml]: Thank you. I fixed it just in case.

**Review by binoue — APPROVED** [review]
mobbed

**Pull request merged**